.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   style_check:
     docker:
-      - image: rainforestapp/circlemator:latest
+      - image: gcr.io/rf-public-images/circlemator:911c4fa3591b4d4b5c92cbbc9ccee275d9d408a1
```

Comment:
This is pointing to the image from this commit on circlemator: https://github.com/rainforestapp/circlemator/commit/911c4fa3591b4d4b5c92cbbc9ccee275d9d408a1 which includes the changes from this PR